Pandas 2.0 breaks test_check_aggregate.expected_fail_return

After the release of pandas 2.0 on the 3rd of April, the unit test test_check_aggregate.expected_fail_return fails with the following output:

```console
==================================== ERRORS ====================================
________________ ERROR collecting tests/test_check_aggregate.py ________________
tests/test_check_aggregate.py:62: in <module>
    ("components", expected_fail_return("")),
tests/test_check_aggregate.py:29: in expected_fail_return
    index = pd.Index(
E   TypeError: __new__() got an unexpected keyword argument 'names'
```

seen for example here: https://github.com/IAMconsortium/nomenclature/actions/runs/4616113288/jobs/8160716594

--- a/tests/test_check_aggregate.py
+++ b/tests/test_check_aggregate.py
@@ -31,7 +31,7 @@ def expected_fail_return(name):
             ("model_a", "scen_a", "World", f"Final Energy{name}", "EJ/yr", 2005),
             ("model_a", "scen_a", "World", "Final Energy|Industry", "EJ/yr", 2005),
         ],
-        names=["model", "scenario", "region", "variable", "unit", "year"],
+        name=("model", "scenario", "region", "variable", "unit", "year"),
     )
     columns = ["variable", "components"]
     return pd.DataFrame([[9.0, 10.0], [8.0, 7.0]], columns=columns, index=index)

